docs/how-to/companion-node-types.md

```diff
@@ -5,77 +5,98 @@ description: Learn about the available node types in IPFS Companion.
+- Use the [IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop) app (for Windows/Linux/Mac), which installs and manages a local IPFS node for you as well as offering an easy, convenient user interface for managing files, peers, and more
 - If you prefer a more hands-on approach:
   - Install IPFS by following the [command line quick-start guide](command-line-quick-start.md)
   - Or run it in [Docker](https://github.com/ipfs/go-ipfs#running-ipfs-inside-docker)
-- If you are using [Brave](https://brave.com/), feel you may want to experiment with _embedded + `chrome.sockets`_ (see below); you can always switch back to _External_ with local IPFS Desktop
+
+Or, if [Brave](https://brave.com/) is your browser of choice, you can use the easiest of all these options: Just enable [Brave's own built-in IPFS node](#provided-by-brave).
 
 ## External
 
-An _external_ node can be any instance of an IPFS daemon that runs outside of a web browser process and exposes _Gateway_ and writable _API_ over HTTP at TCP ports.
+An _external_ node can be any instance of an IPFS daemon that:
+
+- Runs outside of your web browser
+- Exposes a _gateway_ and writeable _API_ over HTTP at TCP ports
 
-At this time, the [go-ipfs](https://github.com/ipfs/go-ipfs) daemon is the preferred implementation. It is easier on CPU, and provides `dhtclient` mode, which
-decreases ambient bandwidth use and smaller battery drain (key characteristics of something that is expected to run in the background all the time).
+At this time, the [go-ipfs](https://github.com/ipfs/go-ipfs) implementation of IPFS is the recommended choice of daemon for running an external node. It is easier on CPU than other implementations, and provides `dhtclient` mode, which decreases ambient bandwidth use and reduces battery drain (key qualities for something that is expected to run in the background all the time).
 
-A good practice is to run it on localhost (`127.0.0.1`), as it provides:
+A good practice is to run your go-ipfs daemon on localhost (`127.0.0.1`), as it provides:
 
 - Increased security (native IPFS used as end-to-end transport)
 - Better UX in the browser (no mixed-content warnings)
-- Improved performance (local loopback is used, no network overhead)
+- Improved performance (local loopback is used, so no network overhead)
+
+As noted above, you can get started with running a go-ipfs node on your local machine in several ways:
+
+- Download [IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop), which installs and manages a local node for you as well as offering an easy, convenient user interface for managing files, peers, and more
+- If you're comfortable with the command line and don't need the convenience of the IPFS Desktop UI, follow the directions in the [command line quick-start guide](command-line-quick-start.md)
+- Docker fans can run and use go-ipfs from [inside a Docker container](https://github.com/ipfs/go-ipfs#running-ipfs-inside-docker)
```

Comment:
```suggestion
- Docker fans can run and use go-ipfs from [inside a Docker container](https://github.com/ipfs/go-ipfs#running-ipfs-inside-docker).
```

We shouldn't link to GitHub readmes if we can help it, but it looks like we don't have this content in docs right now. Issue #586 addresses this.